README.md

```diff
@@ -490,6 +490,51 @@ The available sort tokens are:
 - name
 - path
 
+##### Restrict results and hierarchical values to non-root level
+
+Let's say you have a lot of levels:
+
+```
+- fruits
+  - yellow
+    - citrus
+      - spicy
+```
+
+But you only want to get the values starting at "citrus", you can use `rootPath`
+
+You can specify an root path to filter the hierarchical values
+
+```
+var helper = algoliasearchHelper(client, indexName, {
+  hierarchicalFacets: [{
+    name: 'products',
+    attributes: ['categories.lvl0', 'categories.lvl1', 'categories.lvl2', 'categories.lvl3'],
+    rootPath: 'fruits > yellow > citrus'
+  }]
+});
+
+helper.toggleRefine('products', 'fruits > yellow > citrus'); // Refine on the rootPath
```

Comment:
We can do this internally as proposed by @bobylito.

Also, can you add a throw if the user tries to `toggleRefine()` on a path that is not part of the rootpath?